march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -195,6 +195,7 @@ void MarchHardwareInterface::read(const ros::Time& /* time */, const ros::Durati
     joint.readEncoders(elapsed_time);
     joint_position_[i] = joint.getPosition();
     joint_velocity_[i] = joint.getVelocity();
+    joint_voltage_velocity_[i] = joint.getVoltageVelocity();
```

Comment:
Before assigning something to a vector like this it has to be initialized. You can do this in the `reserveMemory` method. Is this member variable really necessary? Since it does not serve as input to a controller and will probably only be used as input to a Kalman filter right?